protected/humhub/libs/ProfileImage.php

```diff
@@ -104,16 +106,15 @@ public function hasImage()
      * @param String $prefix for the profile image
      * @return String Path to the profile image
      */
-    public function getPath($prefix = "")
+    public function getPath($prefix = '')
     {
         $path = Yii::getAlias('@webroot/uploads/' . $this->folder_images . '/');
 
-        if (!is_dir($path))
-            mkdir($path);
+        FileHelper::createDirectory($path, 0775, true);
```

Comment:
You may ommit 0775 and true because they are the defaults

Reply:
Done